guides/common/attributes-satellite.adoc

```diff
@@ -35,8 +35,8 @@
 :ansible-namespace-example: redhat.satellite._module_name_
 :ansible-namespace: redhat.satellite
 :ansiblefilepath: /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
-:awx: Ansible Tower
-:awx-context: Ansible_Tower
+:awx: Ansible Automation Controller
+:awx-context: Ansible_Automation_Controller
```

Comment:
If we use dashes here, we could use this attribute in host names.

Reply:
I think it's unnecessarily long for host name examples.